_posts/0000-01-02-MondoBazoo.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Life is one big road with lots of signs, 
So when you riding through the ruts, 
Don't you complicate your mind 
Flee from hate, mischief and jealousy 
Don't bury your thoughts; put your vision to reality."

-Bob Marley